### PR TITLE
Emoji for pass fail

### PR DESCRIPTION
**Pull request by taylor**
# re-adding Emoji for pass fail

## Description

- :heavy_check_mark: for PASSED
- :x: for FAILED

fixed double colon typo

## Issues

- #191 

## How has this been tested:
 - [x] Covered by existing integration testing
 - [ ] Added integration testing to cover
 - [ ] Verified all A/C passes
     * [ ] develop
     * [ ] master
     * [x] tag/other branch
 - [x] Test environment
    * [ ] Shared Packet K8s cluster
    * [ ] New Packet K8s cluster
    * [x] Kind cluster
 - [ ] Have not tested

## Types of changes:
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Documentation update

## Checklist:
**Documentation**
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] No updates required.

**Code Review**
- [ ] Does the test handle fatal exceptions, ie. rescue block

**Issue**
- [ ] Tasks in issue are checked off

### CODE REVIEW

**Comment by williscool** [comment]
@taylor 

should rebase this on master now.

also lets make the emojis a variable no?

File: src/tasks/configuration_lifecycle.cr [hunk 1]
```diff
@@ -28,10 +28,10 @@ task "ip_addresses" do |_, args|
     Dir.cd(cdir)
```
Comment (williscool) [inline src/tasks/configuration_lifecycle.cr:28]: lets take the time to move these emojis into a seperate file and use variables for them now

**Comment by taylor** [comment]
> lets take the time to move these emojis into a seperate file and use variables for them now

@williscool how about creating a new PR with those changes?  Same or new feature branch.

**Comment by williscool** [comment]
It's fine to do in a separate Branch and or ticket just want to make sure it gets done

**Pull request merged**